ChangeLog.d/9302.txt

```diff
@@ -0,0 +1,8 @@
+Features
+   * When the new compilation option MBEDTLS_PSA_STATIC_KEY_SLOTS is enabled,
+     key store uses static key slot buffers instead of dynamically allocated
+     ones.
```

Comment:
I'm not sure this is clear to people who don't know the feature. It's saying that static key slots are static and not dynamic, which feels repetitive. Can we make the English description more informative? Hopefully work in that this means the static key store doesn't use malloc?

Reply:
How about something like

```
   * Add new configuration option MBEDTLS_PSA_STATIC_KEY_SLOTS, which
     pre-allocates storage for keys, enabling malloc-less use of key slots.
```

Reply:
I like the concision, but “pre-allocates” is vague. How about “which makes the key store a statically allocated global array, enabling …”?

Reply:
Or just replace "pre-allocates" with "uses static"

```
   * Add new configuration option MBEDTLS_PSA_STATIC_KEY_SLOTS, which
     uses static storage for keys, enabling malloc-less use of key slots.
```